README.md

```diff
@@ -1 +1,65 @@
-# space-x-react-project
\ No newline at end of file
+# Project Name 
+React: Space-x-react-project
+# Description of project.
+In this project, I use a react web application to create a single webpage for space ship.
+
+## Project web demo link.
+Run npm start in an intergrated terminal in order to start a local live server.
+Or try the Live Demo (if available)
+
+## Built With 
+- Html & CSS
+- JavaScript
+- React
+- Visual studio
+- Webpack
+
+# Getting starting 
+To get a local copy up and running follow these simple example steps:
+
+## Prerequisites
+ Vscode. 
+ must have git installed.
+
+### Setup
+- Click on the green button on the top right corner with the text code
+- Click on SSH
+- Copy the text that appears below 
+- Type "git clone" in your terminal and past the copied text next to "git clone" and press ente
+
+## 🛠 Installation & Set Up
+- Click on the green button on the top right corner with the text code
+- Click on "Download ZIP"
+- Extract the downloaded file locally
+
+## 🚀 Building and Running for Production
+
+1. Generate a full static production build
+
+   npm run build
+
```

Comment:
- [ ] [OPTIONAL] It will be better if you update your readme template with an updated version from microverse.